code/modules/projectiles/gun_helpers.dm

```diff
@@ -938,3 +938,19 @@ As sniper rifles have both and weapon mods can change them as well. ..() deals w
 	if(slot != WEAR_L_HAND && slot != WEAR_R_HAND)
 		return FALSE
 	return TRUE
+
+/**
+ * Returns one of the two override values if either are null, preferring the argument value.
+ * Otherwise, returns TRUE if it is in a civilian usable category (Handguns or SMGs), FALSE if it is not.
+ */
+/obj/item/weapon/gun/proc/is_civilian_usable(mob/user, arg_override)
```

Comment:
what's the point of the arg_override arg btw it doesn't seem used

Reply:
it's for the future